.github/workflows/update_google_font.yml

```diff
@@ -0,0 +1,58 @@
+name: 'Update Google Fonts'
+
+on:
+  schedule:
+    # This runs once a week: https://crontab.guru/#0_0_*_*_0
+    - cron: '0 0 * * 0'
+  workflow_dispatch:
+
+env:
+
+jobs:
+  update-fonts:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.NODE_LTS_VERSION }}
+          check-latest: true
+
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 25
+
+      - run: npm i -g pnpm@${PNPM_VERSION}
+
+      - id: get-store-path
+        run: echo STORE_PATH=$(pnpm store path) >> $GITHUB_OUTPUT
+
+      - uses: actions/cache@v3
+        timeout-minutes: 5
+        id: cache-pnpm-store
+        with:
+          path: ${{ steps.get-store-path.outputs.STORE_PATH }}
+          key: pnpm-store-${{ hashFiles('pnpm-lock.yaml') }}
+          restore-keys: |
+            pnpm-store-
+            pnpm-store-${{ hashFiles('pnpm-lock.yaml') }}
+
+      - run: pnpm install
+
+      - run: pnpm run update-google-fonts
+
+        # open a PR using peter-evans/create-pull-request@v5
+      - name: Open pull request
+        uses: peter-evans/create-pull-request@v5
```

Comment:
This seems unnecessary.

Instead can we use the existing octokit context like we do in other repos to create a PR?